R-package/R/lgb.Dataset.R

```diff
@@ -869,6 +870,7 @@ dim.lgb.Dataset <- function(x, ...) {
 #' print(dtrain, verbose = TRUE)
 #' }
 #' @rdname dimnames.lgb.Dataset
+#' @return A list with the dimensioon names of the dataset
```

Comment:
```suggestion
#' @return A list with the dimension names of the dataset
```

Reply:
I really need to clean my keyboard, the stuff stuck under my O key is a problem... :laughing:

Reply:
According to some typos in previews PRs, `t` key also should be examined. 😄